DESCRIPTION

```diff
@@ -3,8 +3,12 @@ Type: Package
 Title: Dirichlet-Multinomial Mixture Model Machine Learning for
         Microbiome Data
 Version: 1.39.0
-Author: Martin Morgan <martin.morgan@roswellpark.org>
-Maintainer: Martin Morgan <martin.morgan@roswellpark.org>
+Authors@R: c(
+          person("Martin", "Morgan", role = "aut"),
```

Comment:
Martin is both the author and maintainer of the package. For his `person` entry. the role should be `role=c("aut", "cre")` and it must also include an email attribute since he is the maintainer as required by Bioconductor standards.